### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by LargePrime** [ShadowTheAge/yafc#198]
- Be sure that you are using the latest version of YAFC
Flatpack downloaded yesterday
- Please note that YAFC is my personal project and I can't promise any support!
Love you!
- Specify the YAFC version
Could you add a way to determine this to these instructions?
https://flathub.org/apps/details/com.github.petebuffon.yafc
says, today, Version 0.5.8  July 25, 2021

- Specify as much info as possible.
uname -a Linux box 6.1.4-artix1-1 #1 SMP PREEMPT_DYNAMIC Tue, 10 Jan 2023 00:30:06 +0000 x86_64 GNU/Linux
Download yafc flatpack, Launch with
flatpak run com.github.petebuffon.yafc
The app cannot engage or see the file system. Cannot create Project, nor set factorio folders.
The app should be able to engage the File system.

- If your bug is related to some mods, attach new game save file so I can sync mods to yours.
Not mod related
- If your bug is related to your .yafc project (for example errors in calculations), attach your .yafc file
Cannot yet make .yafc project

**Comment by veger** [comment]
ShadowTheAge does not provide the flatpack, @petebuffon does (at least he asked a question related to it in #180). On the page you linked it is also stated:
> NOTE: This package is not verified by, affiliated with, or supported by ShadowTheAge, the original creator of YAFC.

No idea if @petebuffon is (still) active in this repo, otherwise you'd need to find another way to contact him.

**Comment by petebuffon** [comment]
@LargePrime go ahead and resubmit your issue at [https://github.com/petebuffon/yafc](https://github.com/petebuffon/yafc).

As veger commented, ShadowTheAge does not provide the flatpak, I do.

**Comment by DaleStan** [comment]
@petebuffon I'm not sure if you're seeing something different than I am, but for me, https://github.com/petebuffon/yafc doesn't have an issues tab, and https://github.com/petebuffon/yafc/issues redirects to https://github.com/petebuffon/yafc/pulls.

**Comment by petebuffon** [comment]
@DaleStan Good catch. I wasn't aware that forked repos don't automatically get an issues section. The issues link works now: [https://github.com/petebuffon/yafc/issues](https://github.com/petebuffon/yafc/issues). Additionally I updated the link at [https://flathub.org/apps/details/com.github.petebuffon.yafc](https://flathub.org/apps/details/com.github.petebuffon.yafc).
